### PR TITLE
Reformats remaining experiment doc md files

### PR DESCRIPTION
**Pull request by asitava1998**
Fixes issue #462 

Changes: 
* All the md files are present as of now have been modified.

Screenshots for the change: 
[app.zip](https://github.com/fossasia/pslab-android/files/1278295/app.zip)

Note:
* Since, there are around 30 files for which markdown docs have been added, it is better to test the apk.
* In some cases, there are images missing. It is not an error due to the code. The screenshots/schematics corresponding to those images have not been created yet.

### CODE REVIEW

**Comment by viveksb007** [comment]
@asitava1998 I tested the apk you provided. `svg` images aren't rendered in markdown view like general images like `png`,`jpg`,etc. Have a look at [this](https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown) stackoverflow post to see how `svg` images could be rendered in markdown image syntax.
You can check experiments like `NFET Output Characteristics` and `NFET Transfer Characteristics` where `svg` images are used.
Thanks

**Comment by asitava1998** [comment]
@viveksb007  It was not a rendering problem. `.svg` files are being rendered properly by MarkdownView. The problem you highlighted was due to a small typo in the image hyperlink. I couldn't notice it because almost in all the cases, images are not rendered because those files do not exist. There re around 50 .md files but only 28 schematics and some 30 odd screenshots. So, some files which are mentioned have not been created yet.
Thanks for highlighting the error though.

**Comment by CloudyPadmal** [comment]
Hi @asitava1998 and @viveksb007 ! We can create those missing svg files for schematics later in a different issue and merge this one with existing resources! What do you guys think? :)

**Comment by asitava1998** [comment]
@CloudyPadmal @viveksb007 I agree with what Padmal said.  We can create a new issue and start creating the svg files for the circuits.

**Comment by CloudyPadmal** [comment]
So we can merge this PR and upload the latest APK to the Play store! :D

**Comment by asitava1998** [comment]
@viveksb007 @CloudyPadmal @akarshan96 Guys, can you review this soon? We can go for the beta release after merging all the PRs.

**Comment by viveksb007** [comment]
@asitava1998 could you provide the updated apk?

**Comment by asitava1998** [comment]
@viveksb007 Here is the updated apk
[app.zip](https://github.com/fossasia/pslab-android/files/1288387/app.zip)

File: app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java [hunk 1]
```diff
@@ -67,93 +67,93 @@ public Fragment getItem(int position) {
                     case "Zener IV Characteristics":
                         return ExperimentDocFragment.newInstance("D_ZenerIV.html");
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java]: This has a md file :)

Comment (asitava1998) [reply app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java]: Fixed :+1:

Comment (CloudyPadmal) [reply app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java]: Awesome! There's a lot of files. So can miss something!

**Comment by asitava1998** [comment]
Thank you @CloudyPadmal

**Pull request merged**